{"links":{"static":"docs/api/fetch-client/interface/Interceptor/property/requestError","html":"docs/api/fetch-client/interface/Interceptor/property/requestError/index.html","fragment":"docs/api/fetch-client/interface/Interceptor/property/requestError/index-fragment.html","self":"docs/api/fetch-client/interface/Interceptor/property/requestError/index.json"},"name":"requestError","kindString":"Property","flags":{"isExported":true,"isOptional":true},"comment":{"shortText":"Handles errors generated by previous request interceptors. This function acts\nas a Promise rejection handler. It may rethrow the error to propagate the\nfailure, or return a new Request or Response to recover.","tags":[{"tag":"param","text":"The rejection value from the previous interceptor.","param":"error"},{"tag":"returns","text":"The existing request, a new request or a response; or a Promise for any of these.\n"}]},"sources":[{"fileName":"dist/aurelia-fetch-client.d.ts","line":27,"character":14}],"type":{"type":"reflection","declaration":{"id":930,"name":"__type","kind":65536,"kindString":"Type literal","flags":{},"signatures":[{"id":931,"name":"__call","kind":4096,"kindString":"Call signature","flags":{},"parameters":[{"id":932,"name":"error","kind":32768,"kindString":"Parameter","flags":{},"type":{"type":"instrinct","name":"any"}}],"type":{"type":"union","types":[{"type":"reference","name":"Request","id":993},{"type":"reference","name":"Response","id":1098},{"type":"reference","name":"Promise","typeArguments":[{"type":"union","types":[{"type":"reference","name":"Request","id":993},{"type":"reference","name":"Response","id":1098}]}]}]}}],"sources":[{"fileName":"dist/aurelia-fetch-client.d.ts","line":27,"character":16}]}}}